plexapi/video.py

```diff
@@ -896,13 +933,47 @@ def _loadData(self, data):
         self.thumbAspectRatio = data.attrib.get('thumbAspectRatio')
         self.viewOffset = utils.cast(int, data.attrib.get('viewOffset', 0))
         self.year = utils.cast(int, data.attrib.get('year'))
-
+        
     @property
     def locations(self):
         """ This does not exist in plex xml response but is added to have a common
             interface to get the locations of the clip.
-
             Retruns:
```

Comment:
```suggestion

            Returns:
```

Reply:
I believe this should address the `sphinx` docstring test failure regarding indentation.